CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Add ZK indexers.
```

Comment:
I think this needs to be a lot more descriptive. Can we document exactly why these were added, and what they will be used for?

Also best if we can include `([#677](https://github.com/hemilabs/heminetwork/pull/677))`